src/main/java/com/minecolonies/coremod/client/gui/WindowCitizen.java

```diff
@@ -209,6 +333,30 @@ public WindowCitizen(final CitizenDataView citizen)
     {
         super(Constants.MOD_ID + CITIZEN_RESOURCE_SUFFIX);
         this.citizen = citizen;
+
+        resourceList = findPaneOfTypeByID(WINDOW_ID_LIST_REQUESTS, ScrollingList.class);
+    }
+
+    /**
+     * Get a nice represetation of the pos string.
+     *
+     * @param pos the position.
+     * @return a nice string.
+     */
+    private static String getNicePositionString(final BlockPos pos)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getNicePositionString" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)